.github/workflows/ci.yml

```diff
@@ -18,7 +18,7 @@ jobs:
       fail-fast: false
       matrix:
         rust:
-          - 1.30.0
+          - 1.36.0
```

Comment:
There *is* no lock file. Can you add a `Cargo.lock.msrv` or so that has versions of everything that would compile with 1.31, switch this here back to 1.31 and for the 1.31 build copy that lock file in place so it's used?

Reply:
I'm unable to generate a lockfile in a format old enough — the oldest Rust that exists for my platform is 1.46.

BTW, 73% of all crates are incompatible with <1.36, 97% of recently updated crates require 1.56+, and 99.5% crates.io requests are from Rust 1.65 or later.

https://lib.rs/stats#rustc-usage

Reply:
Ok, I'll take a look. Don't worry about that :)

I'm aware of those numbers and it's also ridiculous to me that some people can update random crates but not their toolchain, but here we are

Reply:
I don't think there's a single person able to use Rust 1.30 today. This is way below even Debian oldstable.

Even if such user exists, they already must have frozen all their dependencies and won't be able update any of them, so they won't update `pkg-config` either. There's only 6% of crates that work with 1.30 in their latest version, and majority of them are complete junk (reserved names, hello ferris, winapi crates that don't compile with Rust 1.x).

Rust 1.80 broke most versions of the `time` crate, so now Rust is incompatible with most non-trivial projects having `Cargo.lock` compatible with Rust <1.67, and I expect that to push MSRV of most crates even higher.

Reply:
I agree with all you said. If it was for me, I would always just support latest stable. But I don't have the energy anymore to argue with people that have stupid requirements and somehow need a much older version, so if it's easy enough (like here) I rather do that.